package.json

```diff
@@ -657,7 +657,8 @@
     "test": "tsc -p ./test && node ./out/test/runTest.js",
     "coverage": "tsc -p ./test && node ./out/test/runTest.js --coverage",
     "package": "npx vsce package",
-    "format": "prettier --ignore-path .gitignore --write \"**/*.+(js|ts|json)\""
+    "format": "prettier --ignore-path .gitignore --write \"**/*.+(js|ts|json)\"",
+    "publish": "npx vsce publish --yarn"
```

Comment:
Do we want the `--yarn` option here? We're using `npm` elsewhere

Reply:
I just followed the documentation with that, you're right ill remove it and do a dummy release